src/main/java/org/semux/config/TestnetConfig.java

```diff
@@ -31,4 +31,22 @@ public TestnetConfig(String dataDir) {
     public Map<Fork, Long> forkActivationCheckpoints() {
         return Collections.emptyMap();
     }
+
+    /**
+     * Testnet maxes out at 15 validators to stop dead validators from breaking
+     * concensus
+     * 
+     * @param number
+     * @return
+     */
+    @Override
+    public int getNumberOfValidators(long number) {
+
+        // adjust number down to unstick testnet to more reasonable max
+        if (number <= 751638) {
```

Comment:
The hard capped validators won't be written into the database since the database doesn't get updated until block 751800. As a result clients won't be able to sync with the validator nodes with the error of `Not enough votes, needs 2/3+`. See:

https://github.com/semuxproject/semux-core/blob/40a00ccdace0f5ff5fe82b86e9a660f5ca758f39/src/main/java/org/semux/core/BlockchainImpl.java#L381-L384

Reply:
I managed to resume my client using a workaround by calling `updateValidators(751600)` in the constructor of `BlockchainImpl`. A proper solution would be performing a database upgrade.